web-extensions/chrome/engine/messaging.js

```diff
@@ -0,0 +1,59 @@
+
+var BmcWindowHandlers = [];
+
+function BmcMessageHandler(tag, selector, handler) {
+    this.tag = tag;
+    this.select = selector;
+    this.handle = handler;
+}
+
+function BmcMessagingHandler() {
```

Comment:
Is this function supposed to do something different than `BmcMessageHandler`?

Reply:
This is for the mechanism's abstraction huile the one you refer to defines and encapsulates one single handler